packages/video_player/video_player/lib/video_player.dart

```diff
@@ -743,6 +743,12 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   }
 
   void _updatePosition(Duration position) {
+    // The underlying native implementation on some platforms sometimes reports
+    // a position slightly past the reported max duration. Clamp to the duration
+    // to insulate clients from this behavior.
+    if (position.compareTo(value.duration) > 0) {
```

Comment:
nit: Would this not be easier to read as `position > value.duration`.

Reply:
Weird, for some reason I thought that didn't work. Yes, definitely much better that way.